IDEOorg/steps
# #161271325 Implement update media object

#### What does this PR do?
Modify the update functionality for `media` to update only the provided fields

#### Description of Task to be completed?
- modify the update functionality to update only the provided fields
- write a test for the implementation

#### How should this be manually tested?
- Open postman
- Make a `PUT` request to `localhost:3001/api/media/:id`, sending a JSON object of the format:
`{
     "id": 4,
     "task_id": 6,
     "title": "TITLE1",
     "category": "CATEGORY",
     "description": "updated discription",
     "url": "url",
     "image": "image",
     "published_by": 1,
     "type": "GENERAL_EDUCATION"
}`. Feel free to omit any of the fields and send the request
- You should see an updated `media` object as a response

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161271325

#### Screenshots (if appropriate)
<img width="1124" alt="screen shot 2018-10-24 at 9 24 14 pm" src="https://user-images.githubusercontent.com/30550427/47459163-50e42380-d7d3-11e8-8bb2-1f64cbd2c756.png">


#### Questions:
N/A
